docs/PowerAuth-Server-1.6.0.md

```diff
@@ -4,6 +4,23 @@ This guide contains instructions for migration from PowerAuth Server version `1.
 
 ## Database Changes
 
+### Allow Non-personalized Operations
+
+The column `user_id` in table `pa_operation` is nullable now.
+
+#### PostgreSQL
```

Comment:
IMHO the agreement was to only describe the change but not write SQL manually anymore. cc @zcgandcomp

Reply:
I checked the syntax and it should be correct. I will keep the migration guide manually written for now, we can update it next time.